go.mod

```diff
@@ -8,16 +8,17 @@ require (
 	github.com/buger/jsonparser v1.1.1
 	github.com/eventials/go-tus v0.0.0-20211022131811-252c8454f2dc
 	github.com/gobwas/ws v1.1.0
-	github.com/golang-jwt/jwt/v4 v4.1.0
+	github.com/goccy/go-json v0.8.1
+	github.com/golang-jwt/jwt/v4 v4.2.0
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/json-iterator/go v1.1.12
 	github.com/kr/text v0.2.0 // indirect
 	github.com/mitchellh/go-ps v1.0.0
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/qlik-oss/enigma-go/v3 v3.0.1
+	github.com/qlik-oss/enigma-go/v3 v3.0.2-0.20211216135847-b31ee494315e
```

Comment:
Integration version with new goccy/go-json?

Reply:
correct + Float64 unmarshal changes (and some updated dependencies)

Reply:
yes new error message from the part that comes from the used library yes, and yes is less verbose